docs/Contributing.asciidoc

```diff
@@ -333,6 +333,41 @@ Authentication module is expected to return HASH:
 Authentication module is expected to create or update user entry in OpenQA database
 after user validation. See included modules for inspiration.
 
+=== WebAPI Plugins (requires Mojolicious::Plugin::OpenAPI)
+
+At WebUI service startup during configration file processing all sections starting with 
```

Comment:
s/configration/configuration/ but there seem to be more typos, please check spelling in the further document